rotkehlchen/chain/manager.py

```diff
@@ -217,6 +218,13 @@ def __init__(
                         premium=premium,
                         msg_aggregator=msg_aggregator,
                     )
+                elif given_module == 'balancer':
+                    self.eth_modules['balancer'] = Balancer(
+                        ethereum_manager=ethereum_manager,
+                        database=self.database,
+                        premium=premium,
+                        msg_aggregator=msg_aggregator,
```

Comment:
I see you are not using the balances returned by the DeFi SDK. I checked just now and all you can get from there is the amount of BPT tokens. Example:

![2020-10-22_21-59](https://user-images.githubusercontent.com/1658405/96926163-851f7700-14b5-11eb-8a7a-3dcafe92ff83.png)
![2020-10-22_22-00](https://user-images.githubusercontent.com/1658405/96926182-88b2fe00-14b5-11eb-8b67-2013bc1ba1bf.png)

Reply:
At the beginning it was difficult to me understand the whole DeFi SDK/`self.rotkehlchen.chain_manager.defi_balances`. As I said, I even opened the contract, queried Balancer addresses and used the Zerion app.
I finally focused on the subgraph task and omitted any combination of both data. Now it would be more comfortable for be to do that, definitely.